pkgs/development/tools/build-managers/bob/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "bob";
+  version = "0.5.3";
+
+  src = fetchFromGitHub {
+    owner = "benchkram";
+    repo = pname;
+    rev = "${version}";
+    sha256 = "sha256-JG1fysCqqd/MwpNhKJwLr4cTGq4/88f9OMMapb+r3bc=";
+  };
+
+  vendorSha256 = "sha256-R+zXGR5isoo76oc4lsFf9uCM0Kyi8dQiKEg4BUxtv+k=";
+
+  excludedPackages = ["example/server-db" "test/e2e"];
+
+  doCheck = false;
```

Comment:
Please add a comment on why the tests are failing/ disabled.

Reply:
The tests are failing for the following reason:

```shell
panic: failed to make git command: unable to find git executable in PATH; please install git before retrying
```

The code is using git at some point and is not found in PATH.  I wonder if I should add a comment about that or there is some way to make Git available.

Reply:
You can add `git` to `buildInputs`. If the tests requires network access then it won't work, and then just add a comment stating `tests require network access`. Otherwise you may also need `leaveDotGit` or `deepClone`. Search in nixpkgs with those keywords for various usage examples.